### PR TITLE
Remove opcode fields from COPYENTRY as it isn't used, except

### PR DESCRIPTION
**Pull request by jaykrell**
for a self-check.

### CODE REVIEW

File: src/disasm.cpp [hunk 1]
```diff
@@ -886,542 +885,542 @@ const BYTE CDetourDis::s_rbModRm[256] = {
 
 const CDetourDis::COPYENTRY CDetourDis::s_rceCopyTable[257] =
```
Comment (jaykrell) [inline src/disasm.cpp]: How about we remove this explicit 257, remove the terminal entry, and C_ASSERT that the size is 256? That gives some of the sanity check, w/o requiring the tedious recording of the numbers, aside from comments which have less stringent requirement.

**Comment by dtarditi** [comment]
This seems good.  There are some merge conflicts that need to be resolved first.

**Comment by dtarditi** [comment]
@jaykrell, could you resolve the merge conflicts?

**Comment by dtarditi** [comment]
@jaykrell, gentle reminder: could you resolve the merge conflicts?

**Comment by jaykrell** [comment]
I redid the change fairly mechanically, with regexp search/replace, mostly, rather than resolve the conflict.
This should retain confidence.

**Comment by bgianfo** [comment]
@JayKrell can you rebase this on master again, so we can get CI build validation?

**Comment by bgianfo** [comment]
@dtarditi You ok with merging this now?

**Pull request merged**